backend/bang-ggood/src/main/java/com/bang_ggood/checklist/controller/ChecklistController.java

```diff
@@ -0,0 +1,26 @@
+package com.bang_ggood.checklist.controller;
+
+import com.bang_ggood.checklist.dto.ChecklistCreateRequest;
+import com.bang_ggood.checklist.service.ChecklistService;
+import jakarta.validation.Valid;
+import java.net.URI;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class ChecklistController {
+
+    private final ChecklistService checklistService;
+
+    public ChecklistController(ChecklistService checklistService) {
+        this.checklistService = checklistService;
+    }
+
+    @PostMapping("/checklists")
+    public ResponseEntity<Void> createChecklist(@Valid @RequestBody ChecklistCreateRequest checklistCreateRequest) {
+        long checklistId = checklistService.createChecklist(checklistCreateRequest);
+        return ResponseEntity.created(URI.create("checklists/" + checklistId)).build();
```

Comment:
사소한 부분이지만 API문서에는 `/checklists/{id}`로 나와있어서 통일시켜주면 좋을 것 같아요
```suggestion
        return ResponseEntity.created(URI.create("/checklists/" + checklistId)).build();
```

Reply:
반영했습니닷~